src/libraries/System.Reflection.MetadataLoadContext/tests/src/TestUtils/CoreMetadataAssemblyResolver.cs

```diff
@@ -22,7 +23,17 @@ public override Assembly Resolve(MetadataLoadContext context, AssemblyName assem
                 name.Equals("System.Runtime.InteropServices", StringComparison.OrdinalIgnoreCase))
             {
                 if (_coreAssembly == null)
+                {
                     _coreAssembly = context.LoadFromStream(TestUtils.CreateStreamForCoreAssembly());
+                    //if (PlatformDetection.IsNotBrowser)
```

Comment:
Why is this commented out? Do we need these comments anymore?

Reply:
I need to change bad habits 😜.